tests/functional/application-packages/Execute_WorkflowFile_To_SubsetCRIM.json

```diff
@@ -0,0 +1,32 @@
+{
+  "mode": "async",
+  "response": "document",
+  "inputs": [
+    {
+      "id": "file",
+      "href": "https://github.com/Ouranosinc/xclim/raw/master/tests/testdata/HadGEM2-CC_360day/tasmax_day_HadGEM2-CC_rcp85_r1i1p1_na10kgrid_qm-moving-50bins-detrend_2095.nc"
```

Comment:
use a commit id so that we keep the same reference, so avoid suddenly changing test result if this is modified